rosie/rosie/chamber_of_deputies/adapter.py

```diff
@@ -112,6 +112,6 @@ def rename_categories(self, df):
         df['is_party_expense'] = df['congressperson_id'].isnull()
 
     def coerce_dates(self, df):
-        for field in ('issue_date', 'situation_date'):
+        for field, fmt in (('issue_date', None), ('situation_date', '%d/%m/%Y')):
```

Comment:
Is there a specific reason for this `None` here? I think _explicit is better than implicit_ so if we're going with the default, may we have the default value instead of just `None`?

Reply:
How I can do this?

Reply:
Well, As we discussed internally, I will expand this PR to also consider issue_date.